Dockerfile.spyre

```
@@ -14,17 +14,12 @@ RUN microdnf update -y && microdnf install -y \
     && microdnf clean all
 
 # Download and install vllm ###########################################################
-ENV VLLM_TARGET_DEVICE=empty
-RUN git clone -b sop-plugin-refactoring --single-branch https://github.com/IBM/vllm.git \
-    && cd vllm \
-    && python3.12 -m pip install --upgrade pip \
-    && pip install -r requirements-build.txt \
-    && pip install --no-build-isolation -v -e . \
-    && mkdir /workspace/vllm-spyre
+RUN pip${PYTHON_VERSION} install vllm==0.7.3
```

I suspect this is why spyre tests are still failing
```suggestion
RUN pip${PYTHON_VERSION} install --upgrade pip && pip install vllm==0.7.3
```